app/models/chargeback_container_project.rb

```diff
@@ -42,20 +42,20 @@ def self.build_results_for_report_ChargebackContainerProject(options)
     project_id = options[:entity_id]
     filter_tag = options[:tag]
 
-    @project_ids = if filter_tag.present?
-                     # Get all ids of tagged projects
-                     ContainerProject.find_tagged_with(:all => filter_tag, :ns => "*").pluck(:id)
-                   elsif provider_id == "all"
-                     ContainerProject.all.pluck(:id)
-                   elsif provider_id.present? && project_id == "all"
-                     ContainerProject.where('ems_id = ? or old_ems_id = ?', provider_id, provider_id).pluck(:id)
-                   elsif project_id.present?
-                     [project_id.to_i]
-                   elsif project_id.nil? && provider_id.nil? && filter_tag.nil?
-                     raise "must provide option :entity_id, provider_id or tag"
-                   end
+    @projects = if filter_tag.present?
```

Comment:
@zeari I couldn't find the need to convert this to objects from ids. Can you point me where this is needed?